src/renderer/utils/invite.ts

```diff
@@ -0,0 +1,7 @@
+const INVITE_URL_HOST = 'i.delta.chat'
+
+/** Converts an invite QR code to an HTTP invite URL */
+export function qrCodeToInviteUrl(qrData: string): string {
+  const code = qrData.split(':', 2)[1].replace('#', '&')
```

Comment:
[double checked](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) as i am always confused by that: replace() only replaces the first occurrence, fine :)